packages/sdks/snippets/hydrogen/app/routes/landing-page.tsx

```diff
@@ -0,0 +1,31 @@
+import {fetchOneEntry} from '@builder.io/sdk-react';
+import {NavBar} from '../components/NavBar';
+import {type LoaderFunction} from '@remix-run/server-runtime';
+import {useLoaderData} from '@remix-run/react';
+
+export const loader: LoaderFunction = async () => {
+  const content = await fetchOneEntry({
```

Comment:
you've used `links` as variable name for all of the other snippets and I think that's more accurate here right?